composer.json

```diff
@@ -10,7 +10,8 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9"
+        "php": ">=5.5.9",
+        "laravel/installer": "^1.3"
```

Comment:
@quickliketurtle awesome--would you mind updating this for 1.3.1 or greater, and then we can release?